src/raftstore/store/store.rs

```diff
@@ -1264,6 +1266,8 @@ impl<T: Transport, C: PdClient> Store<T, C> {
                 continue;
             }
 
+            total_gc_logs += compact_idx - first_idx;
```

Comment:
If restart, it may be inaccurate at first, because we updated the first index before, but have not truncated the logs in the worker.